lib/aiken/collection/list.ak

```diff
@@ -136,6 +136,8 @@ pub fn at(self: List<a>, index: Int) -> Option<a> {
     [x, ..xs] ->
       if index == 0 {
         Some(x)
+      } else if index < 0 {
```

Comment:
It would be better for this check to live outside of the loop / recursion. Otherwise we pay the cost at each step.